tests/rptest/tests/acls_test.py

```diff
@@ -543,11 +550,25 @@ def test_describe_acls(self, use_tls: bool, use_sasl: bool,
         """
         security::acl_operation::describe, security::default_cluster_name
         """
-        self.prepare_cluster(use_tls,
-                             use_sasl,
-                             enable_authz,
-                             authn_method,
-                             client_auth=client_auth)
+        def expect_startup_failure(use_tls: bool, authn_method: Optional[str],
+                                   client_auth: bool):
+            if authn_method == 'mtls_identity':
```

Comment:
nit: I'm surprised this line does not fail when `authn_method=None`

Reply:
Is there something more idiomatic I should do?

Reply:
Hm. I tested and it's fine. I think I was confusing C++ and Python.

Reply:
It works in C++, too.